ts/Series/Treegraph/TreegraphPoint.ts

```diff
@@ -0,0 +1,86 @@
+/* *
+ *
+ *  (c) 2010-2022 Pawel Lysy Grzegorz Blachlinski
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+/* *
+ *
+ *  Imports
+ *
+ * */
+
+import type TreegraphPointOptions from './TreegraphPointOptions';
+import type TreegraphSeries from './TreegraphSeries';
+import type TreegraphNode from './TreegraphNode';
+import type TreegraphLink from './TreegraphLink';
+import SeriesRegistry from '../../Core/Series/SeriesRegistry.js';
+
+const {
+    seriesTypes: {
+        treemap: {
+            prototype: {
+                pointClass: TreemapPoint
+            }
+        }
+    }
+} = SeriesRegistry;
+
+
+/* *
+ *
+ *  Class
+ *
+ * */
+
+class TreegraphPoint extends TreemapPoint {
+    public options: TreegraphPointOptions = void 0 as any;
+    public isLink = false;
+    public series: TreegraphSeries = void 0 as any;
+    public collapsed: boolean = false;
+    public node: TreegraphNode.Node = void 0 as any;
+    public level?: number;
+    public linkToParent?: TreegraphLink;
+    hidden: boolean = false;
```

Comment:
We already have the [Point.visible](https://github.com/highcharts/highcharts/blob/338528d0b241d201eb7597cb20244fababe10ea2/ts/Core/Series/Point.ts#L237) property. Do we need to add another property that seems to do exactly the same, but with a negated sign?

Reply:
You're right. Deleted and replaced :)